mkdocs/reference/functions/index.md

```diff
@@ -0,0 +1,4 @@
+# Functions
+## What Can Functions Do For You?
+Functions enable you to connect your Visivo configurations, create re-usable object, inject context and produce different outputs in different environments. 
```

Comment:
What do yo mean "connect your Visivo configurations"?

Reply:
I was talking about how ref connects objects together in the dag